wren-ai-service/src/core/engine.py

```diff
@@ -31,8 +31,15 @@ def clean_generation_result(result: str) -> str:
     def _normalize_whitespace(s: str) -> str:
         return re.sub(r"\s+", " ", s).strip()
 
+    def _remove_invalid_json_chars(s: str) -> str:
```

Comment:
im wondering to know what scenario you encountered. could you pass me some information about it?